src/pages/journal/journal.analytics.effects.ts

```diff
@@ -0,0 +1,121 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { of } from 'rxjs/observable/of';
+import { switchMap } from 'rxjs/operators';
+// import { Store} from '@ngrx/store';
+// import { StoreModel } from '../../common/store.model';
+// import { getJournalState } from '../journal/journal.reducer';
+import { AnalyticsProvider } from '../../providers/analytics/analytics';
+import {
+  JOURNAL_VIEW_DID_ENTER,
+  JOURNAL_NAVIGATE_DAY,
+  JournalNavigateDay,
+  JOURNAL_REFRESH,
+  JournalRefresh,
+  JOURNAL_REFRESH_ERROR,
+  JournalRefreshError } from '../../pages/journal/journal.actions';
+import {
+    AnalyticsDimensionIndices,
+    AnalyticsScreenNames,
+    AnalyticsEventCategories,
+    AnalyticsEvents,
+  } from '../../providers/analytics/analytics.model';
+import { SLOT_HAS_CHANGED, SlotHasChanged } from '../../providers/slot/slot.actions';
+import { TEST_OUTCOME_START_TEST, TestOutcomeStartTest } from './components/test-outcome/test-outcome.actions';
+
+@Injectable()
+export class JournalAnalyticsEffects {
+
+  constructor(
+    public analytics: AnalyticsProvider,
+    private actions$: Actions,
+//    private store$: Store<StoreModel>,
```

Comment:
Wonder if there is a linting rule that can check for these

Reply:
I doubt it. But I will watch out for it in future :)  
Removed